Elvira Bruno - Projeto Guiado II - Semana 10

elvirabl opened the pull request:
**PROJETO GUIADO API**
_Banco de dados de um banco_

- Buscas por nome e CPF, usando o método GET
- Busca exclusiva por ID usando o método GET e função MAP
- Através do método PATH, podemos fazer a alteração dos dados cadastrais de um cliente existente.
- O serviço de Saque, pode ser efetuado a partir de um cliente existente, utilizando método PATCH 
- A adição de um novo cliente com um ID aleatório, será efetuada através do método POST com o auxílio da ferramenta UUID.
- O método DELETE foi usado para excluir um cliente do banco de dados, através do ID exclusivo do cliente.

- Aguardo apontamentos da professora.

elvirabl committed:
Elvira Bruno - Projeto Guiado II - Semana 10

manuellysuzik commented on exercicios/projeto-casa/index.js:
```diff
@@ -0,0 +1,133 @@
+    const clienteFiltrado = listaClientes.filter((item) => {
+
+        if (filtroNome) {
+            return item.nome_cliente.toLowerCase() === filtroNome.toLowerCase()
```
`filtroNome` já foi convertido pra `toLowerCase()`

manuellysuzik replied:
filtroNome já foi convertido pra toLowerCase()

manuellysuzik commented on exercicios/projeto-casa/index.js:
```diff
@@ -0,0 +1,133 @@
+
+app.get("/contas-clientes/:id", (req, res) => {
+    const id = req.params.id
+    const clienteEscolhido = listaClientes.find((item, index) => cliente.id == id)
```
isso nunca vai ser encontrado pois você declarou como item e usou cliente.

elvirabl committed:
Alteração - Linhas 8, 9, 10

Conforme apontamento da professora, linhas alteradas de params para query.
Obrigada
